#CREATE request parameters
int8 mode
string[] names
float32[] angles
float32 max_speed
float32[] time_second

float32 mask
geometry_msgs/Twist target

bool flag
--------------------------------
#CREATE response parameters
geometry_msgs/Twist current